test/index.test.ts

```diff
@@ -1,11 +1,8 @@
 import * as assert from 'assert';
 import * as extend from 'extend';
 import * as proxyquire from 'proxyquire';
+import * as _gcpMetadata from '../src';
```

Comment:
nit: I think you can drop the underscore prefix now.

Reply:
Handling that in the follow-up PR #20 :)